Assets/MixedRealityToolkit.Services/InputSimulation/SimulatedHand.cs

```diff
@@ -117,6 +118,7 @@ public void UpdateState(SimulatedHandData handData)
                 }
                 else
                 {
+                    jointPositions[i] = Vector3.SmoothDamp(jointPoses[handJoint].Position, jointPositions[i], ref jointPositionVelocities[i], 0.15f);
```

Comment:
The smoothTime should be configurable, not least because we want to be able to disable it for testing purposes. The value should go into the input simulation profile, with a matching entry in the inspector, like so:

```diff
diff --git a/Assets/MixedRealityToolkit.Services/InputSimulation/Editor/MixedRealityInputSimulationProfileInspector.cs b/Assets/MixedRealityToolkit.Services/InputSimulation/Editor/MixedRealityInputSimulationProfileInspector.cs
index 60001148d..17b62a8f3 100644
--- a/Assets/MixedRealityToolkit.Services/InputSimulation/Editor/MixedRealityInputSimulationProfileInspector.cs
+++ b/Assets/MixedRealityToolkit.Services/InputSimulation/Editor/MixedRealityInputSimulationProfileInspector.cs
@@ -47,6 +47,7 @@ public class MixedRealityInputSimulationProfileInspector : BaseMixedRealityToolk
         private SerializedProperty defaultHandDistance;
         private SerializedProperty handDepthMultiplier;
         private SerializedProperty handJitterAmount;
+        private SerializedProperty handSmoothTime;

         private SerializedProperty yawHandCWKey;
         private SerializedProperty yawHandCCWKey;
@@ -104,6 +105,7 @@ protected override void OnEnable()
             defaultHandDistance = serializedObject.FindProperty("defaultHandDistance");
             handDepthMultiplier = serializedObject.FindProperty("handDepthMultiplier");
             handJitterAmount = serializedObject.FindProperty("handJitterAmount");
+            handSmoothTime = serializedObject.FindProperty("handSmoothTime");

             yawHandCWKey = serializedObject.FindProperty("yawHandCWKey");
             yawHandCCWKey = serializedObject.FindProperty("yawHandCCWKey");
@@ -190,6 +192,7 @@ public override void OnInspectorGUI()
                 EditorGUILayout.PropertyField(defaultHandDistance);
                 EditorGUILayout.PropertyField(handDepthMultiplier);
                 EditorGUILayout.PropertyField(handJitterAmount);
+                EditorGUILayout.PropertyField(handSmoothTime);
                 EditorGUILayout.Space();

                 EditorGUILayout.PropertyField(yawHandCWKey);
diff --git a/Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs b/Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs
index 6a4d7564b..b441fb7f1 100644
--- a/Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs
+++ b/Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs
@@ -151,6 +151,10 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
         [Tooltip("Apply random offset to the hand position")]
         private float handJitterAmount = 0.0f;
         public float HandJitterAmount => handJitterAmount;
+        [SerializeField]
+        [Tooltip("Time it takes hands to reach the target mouse position")]
+        private float handSmoothTime = 0.15f;
+        public float HandSmoothTime => handSmoothTime;

         [Header("Hand Rotation Settings")]
         [SerializeField]
```